doc/rados/operations/crush-map-edits.rst

```diff
@@ -32,193 +33,199 @@ Pool Values`_.
 
 .. _getcrushmap:
 
-Get a CRUSH Map
----------------
+Get the CRUSH Map
+-----------------
 
-To get the CRUSH map for your cluster, execute the following:
+To get the CRUSH map for your cluster, run a command of the following form:
 
 .. prompt:: bash $
 
-	ceph osd getcrushmap -o {compiled-crushmap-filename}
+    ceph osd getcrushmap -o {compiled-crushmap-filename}
 
-Ceph will output (-o) a compiled CRUSH map to the filename you specified. Since
-the CRUSH map is in a compiled form, you must decompile it first before you can
-edit it.
+Ceph outputs (``-o``) a compiled CRUSH map to the filename that you have
+specified. Because the CRUSH map is in a compiled form, you must first
+decompile it before you can edit it.
 
 .. _decompilecrushmap:
 
-Decompile a CRUSH Map
----------------------
+Decompile the CRUSH Map
+-----------------------
 
-To decompile a CRUSH map, execute the following:
+To decompile the CRUSH map, run a command of the following form:
 
 .. prompt:: bash $
 
-	crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
+    crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
```

Comment:
On line 17 there's a step to edit the decompiled map, but no corresponding elaboration below at this point.  Suggest something succinct using one's favorite *text* editor.  And maybe an admonition to not use something like MS Word or Interleaf.

Reply:
I've added a directive that the reader should use a text editor.